cypress/unit/store-getter-getCategoricalJsonOutput.cy.js

```diff
@@ -0,0 +1,106 @@
+import { getters } from "~/store";
+
+const category = "category1";
+const columnName = "column1";
+let store = {
+
+    getters: getters,
+
+    state: {
+
+        categoricalOptions: {
+
+            category1: [
+
+                { label: "annotatedValue1", identifier: "https://example.org/option_1"},
+                { label: "annotatedValue2", identifier: "https://example.org/option_2"}
+            ]
+        },
+        categories: {
+            "category1": {
+
+                componentName: "annot-categorical",
+                explanation: "This is an explanation for how to annotate category1.",
+                identifier: "nb:hasCategory1"
+            }
+        },
+        columnToCategoryMap: { column1: "category1" },
+        dataDictionary: {
+
+            annotated: {
+
+                column1: {
+
+                    valueMap: { "rawValue1": "https://example.org/option_1" },
+                    missingValues: {0: "missing"}
+                }
+            }
+        }
+    }
+};
+
+describe("getCategoricalJsonOutput", () => {
+
+    it("Make sure categorical json output is schema compliant", () => {
+
+        // Act - Get formatted json output data for discrete value column
+        const output = store.getters.getCategoricalJsonOutput(store.state)(columnName);
+
+        // Assert - Current annotated data dictionary schema compliance
+
+        // Parent level
+
+        expect(output).to.have.property("Annotations");
```

Comment:
:cherries:: I find the following section quite hard to read and understand. I think it'd be clearer to just write out the expected json and then `deep.equal` it (assuming that would work).

Reply:
Something like this?
```Javascript
expect(output).to.deep.equal(
    {

        Annotations: {

            IsAbout: {

                Label: "category1",
                TermURL: "nb:hasCategory1"
            },
            Levels: {
                rawValue1: {
                    Label: "annotatedValue1",
                    TermURL: "https://example.org/option_1"
                }
            },
            MissingValues: ["missing"]
        }
})
```
This does work.

Reply:
I find this much clearer, yes, because I can immediately understand what the passed data looks like.